conda_rpms/install.py

```diff
@@ -546,6 +777,30 @@ def link(pkgs_dir, prefix, dist, linktype=LINK_HARD, index=None, target_prefix=N
                 log.error('failed to link (src=%r, dst=%r, type=%r, error=%r)' %
                           (src, dst, lt, e))
 
+        # noarch package specific installation steps
+        if noarch == 'python':
```

Comment:
@lbdreyer If you agree with the above [comment](https://github.com/SciTools-incubator/conda-rpms/pull/9/files#r178893221) then it might be best if this was just:
```python
if noarch:
```